aqua/kitty/Portfile

```diff
@@ -33,7 +33,7 @@ if {${os.major} <= 15} {
     }
 } elseif {${os.major} <= 19} {
     # See https://github.com/kovidgoyal/kitty/issues/5556
-    github.setup        kovidgoyal kitty 0.26.2 v
+    github.setup        kovidgoyal kitty 0.30.0 v
```

Comment:
Please, rebase the PR from the upstream repository, as the master already has this.